docs/Background/Acquisition/Acquisition.md

```diff
@@ -11,7 +11,7 @@
 ## Contracting Officer's Representative (COR)
 
 ### OFA COR
-[OFA COR Delegation Letter](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Acquisition/COR%20Designation%20Letter%20Lauren%20Frohlich%20-%20TANF.pdf)
+[OFA COR Delegation Letter](https://github.com/raft-tech/TANF-app/blob/develop/docs/Acquisition/COR%20Designation%20Letter%20Lauren%20Frohlich%20-%20TANF.pdf)
```

Comment:
@abottoms-coder thanks for taking the lead on getting the relative links fixed. should this continue throughout for this PR?

Reply:
I didn't want to take ownership of updating EVERYTHING -- there's a lot.

Reply:
we'll continue this work in follow-up PRs.